src/index.html

```diff
@@ -59,33 +62,19 @@
     </header>
 
     <main class="catalog">
-      <div class="catalog__card">
-        Product card
-      </div>
+      <div class="catalog__card">Product card</div>
 
-      <div class="catalog__card">
```

Comment:
These changes are optional, but I think it was easier to read as it was before. There were no requirements to fix the code style.

Reply:
I think that it`s Prettier work. I forgot to disabled it for this project